src/Catty.xcodeproj/project.pbxproj

```diff
@@ -7427,6 +7439,14 @@
 			name = ListBricks;
 			sourceTree = "<group>";
 		};
+		BCC25E28231A874200DB639C /* Recovered References */ = {
```

Comment:
Please have a look at the SpriteBubbleConstraintsTests.swift integration in the project file.